src/usng.ts

```diff
@@ -638,15 +638,27 @@ extend(Converter.prototype, {
     }
   },
 
+  isInUPSSpace(lat) {
+    return lat > 84.0 || lat < -80.0
+  },
+
   LLtoUTMUPSObject(lat, lon) {
     // sanity check on input - turned off when testing with Generic Viewer
     if (lon > 180 || lon < -180 || lat > 90 || lat < -90) {
       throw new Error(`usng.js, LLtoUTMUPS, invalid input. lat: ${lat.toFixed(4)} lon: ${lon.toFixed(4)}`);
     }
-    // Constrain reporting UTM coords to the latitude range [80S .. 84N]
-    return (lat > 84.0 || lat < -80.0)
-      ? { ...this.LLtoUPS(lat, lon), zoneNumber: 0 }
-      : { ...this.LLtoUTM(lat, lon), northPole: lat >= 0 };
+    if (this.isInUPSSpace(lat)) {
+      return { ...this.LLtoUPS(lat, lon), zoneNumber: 0 }
+    } else {
+      let utmcoords = [0, 0, 0, 'N']
+      this.LLtoUTMwithNS(lat, lon, utmcoords)
+      return {
```

Comment:
This confused me a bit at first because I didn't realize utmcoords would be mutated by LLtoUTMwithNS.  

Probably my own preference here, but I would prefer to avoid the mutation and have LLtoUTMwithNS return something instead.  

That's outside this change though.